html-block-and-inline/html-block-and-inline-quiz-notes.md

```diff
@@ -0,0 +1,49 @@
+# html-block-and-inline-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- How do block-level elements affect the document flow?
+
+Starts on a new line
+
+- How do inline elements affect the document flow?
+
+Do not start on a new line
+
+- What are the default width and height of a block-level element?
+
+The full width and height available
```

Comment:
The entire width of the parent container & as much height as the content takes up